cli/src/base_cli/commands/balance.rs

```diff
@@ -31,11 +31,7 @@ impl BalanceCommand {
 	pub(crate) fn run(&self, cli: &Cli) -> CliResult {
 		let api = get_chain_api(cli);
 		let accountid = get_accountid_from_str(&self.account);
-		let account_data = if let Some(data) = api.get_account_data(&accountid).unwrap() {
-			data
-		} else {
-			AccountData::default()
-		};
+		let account_data = api.get_account_data(&accountid).unwrap().unwrap_or_default();
```

Comment:
We have changed the implementation. It returns now, our `AccountData` instead of the one from the api-client. This simplifies some things.